src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,23 @@
+import { jwtService } from '../services/jwt.service.js';
+
+export const authMiddleware = (req, res, next) => {
+  const authorization = req.headers['authorization'] || '';
+  const [, token] = authorization.split(' ');
+
+  if (!authorization || !token) {
+    res.status(401).send('Token is not valid');
+
+    return;
+  }
+
+  const userData = jwtService.verify(token);
+
+  if (!userData) {
+    res.status(401).send('Token is not valid');
```

Comment:
The error message 'Token is not valid' is repeated. You might want to create a utility function for sending error responses to avoid repetition and ensure consistency.